packages/fether-react/src/RequireHealthOverlay/HealthModal/HealthModal.js

```diff
@@ -36,47 +39,47 @@ class HealthModal extends Component {
   }
 
   renderTitle = () => {
-    const { healthStatus } = this.props;
+    const { status } = this.props;
 
-    switch (healthStatus) {
-      case STATUS.NO_NODE_CONNECTED_AND_NO_INTERNET:
-        return 'No internet. No nodes connected';
-      case STATUS.DOWNLOADING:
-        return 'Downloading Parity Ethereum...';
-      case STATUS.LAUNCHING:
-        return 'Launching the node...';
-      case STATUS.NODE_CONNECTED_AND_NO_INTERNET:
-        return 'No internet. Connected to node';
-      case STATUS.NO_CLOCK_SYNC:
-        return 'Clock of host not in sync';
-      case STATUS.NO_PEERS:
-        return 'No peer node connections';
-      case STATUS.SYNCING:
-        return 'Syncing...';
-      default:
-        return '';
+    if (!status.nodeConnected && !status.internet) {
+      return 'No internet. No nodes connected';
+    } else if (status.downloading) {
+      return 'Downloading Parity Ethereum...';
+    } else if (status.launching) {
+      return 'Launching the node...';
+    } else if (status.nodeConnected && !status.internet) {
+      return 'No internet. Connected to node';
+    } else if (!status.clockSync) {
+      return 'Clock of host not in sync';
+    } else if (!status.peers) {
+      return 'No peer node connections';
+    } else if (status.syncing) {
+      return `Syncing...`;
+    } else {
+      return '';
     }
   };
 
   renderDescription = () => {
-    const { healthPercentage, healthStatus } = this.props;
+    const { chainName, payload, status } = this.props;
+
+    const chainNameAppend = isLoading(chainName) ? '' : ` (${chainName})`;
```

Comment:
prefer `.pipe(withoutLoading())` in the HOC, which is basically a shorthand for this.

Reply:
I've pushed a commit addressing this comment